libs/@guardian/react-crossword/src/components/Layout.tsx

```diff
@@ -2,28 +2,11 @@ import { css } from '@emotion/react';
 import { headlineBold17, space } from '@guardian/source/foundations';
 import { textSans12, textSans14 } from '@guardian/source/foundations';
 import type { ReactNode } from 'react';
-import { useMemo } from 'react';
 import { memo } from 'react';
 import type { Direction } from '../@types/Direction';
-import { useData } from '../context/Data';
 import { useTheme } from '../context/Theme';
-
-const useGridWidth = () => {
-	const { gutter, cellSize } = useTheme();
-	const { dimensions } = useData();
-
-	return useMemo(
-		() => Math.max((cellSize + gutter) * dimensions.cols + gutter, 300),
-		[cellSize, gutter, dimensions.cols],
-	);
-};
-
-const useWidthForCols = (cols: number) => {
-	const gridWidth = useGridWidth();
-	const { clueMinWidth } = useTheme();
-
-	return gridWidth + clueMinWidth * cols + 'px';
-};
+import { useGridWidth } from '../hooks/useGridWidth';
+import { useWidthForCols } from '../hooks/useWidthForCols';
```

Comment:
are these needed elsewhere now?

Reply:
Not at the moment but i they were at one point in this branch and i liked having them separate